sdk/storage/azure-storage-blob/samples/blob_samples_common_async.py

```diff
@@ -10,43 +10,21 @@
 import asyncio
 from azure.core.exceptions import ResourceExistsError
 
-try:
-    import settings_real as settings
-except ImportError:
-    import blob_settings_fake as settings
-
-from testcase import (
-    StorageTestCase,
-    TestMode,
-    record
-)
 
+# set up
 SOURCE_FILE = 'SampleSource.txt'
+data = b"Lorem ipsum dolor sit amet, consectetur adipiscing elit"
+with open(SOURCE_FILE, 'wb') as stream:
+    stream.write(data)
 
 
-class TestCommonBlobSamplesAsync(StorageTestCase):
-
-    connection_string = settings.BLOB_CONNECTION_STRING
-
-    def setUp(self):
-        data = b"Lorem ipsum dolor sit amet, consectetur adipiscing elit"
-        with open(SOURCE_FILE, 'wb') as stream:
-            stream.write(data)
-
-        super(TestCommonBlobSamplesAsync, self).setUp()
-
-    def tearDown(self):
-        if os.path.isfile(SOURCE_FILE):
-            try:
-                os.remove(SOURCE_FILE)
-            except:
-                pass
+class CommonBlobSamplesAsync(object):
 
-        return super(TestCommonBlobSamplesAsync, self).tearDown()
+    connection_string = os.getenv(CONNECTION_STRING)
 
     #--Begin Blob Samples-----------------------------------------------------------------
 
-    async def _test_blob_snapshots_async(self):
+    async def blob_snapshots_async(self):
```

Comment:
Observation; having both sync and async samples in the same file means that the sample cannot be run on Python 2.

Reply:
good catch...we have a different files right now..so i think that'a good.